Sources/Cache/DiskStorage.swift

```diff
@@ -116,7 +116,14 @@ public enum DiskStorage {
                 // The estimated expiration date.
                 .modificationDate: expiration.estimatedExpirationSinceNow.fileAttributeDate
             ]
-            config.fileManager.createFile(atPath: fileURL.path, contents: data, attributes: attributes)
+            let fileWrote = config.fileManager.createFile(
+                atPath: fileURL.path, contents: data, attributes: attributes
+            )
+            if !fileWrote {
```

Comment:
Guard would seem more fitting to me.
Otherwise it looks great to me

Reply:
As the last statement of this function, it is in fact not guarding a condition but check the result. So I prefer using `if` here.